test/system/120-load.bats

```diff
@@ -123,7 +123,7 @@ verify_iid_and_name() {
     run_podman image scp -q ${notme}@localhost::$newname
 
     expect="Loaded image: $newname"
-    is "$output" "$expect" "-q silences output"
+    assert "$output" =~ "$expect" "-q silences output"
```

Comment:
No, please do not do that. Please fix the scp bug instead. If all scp users start to see this it is pretty bad.

Reply:
Hmm, its not part of this PR then. Let keep this PR on hold. I am creating a new PR to fix scp issue.